tech.json

```diff
@@ -522,31 +546,22 @@
               ],
```

Comment:
I do wonder if spring wall should require tricky spring ball jump

Reply:
Its a tough question.  I don't think `canSpringwall` is useful as a tech at that point.  But a lot of the time you need to delay the springballjump too.